libraries/AP_NavEKF3/AP_NavEKF3_core.h

```diff
@@ -1306,4 +1317,9 @@ class NavEKF3_core : public NavEKF_core_common
 
     // vehicle specific initial gyro bias uncertainty
     float InitialGyroBiasUncertainty(void) const;
+
+    // The following declarations are used to control when the main navigation filter resets it's yaw to the estimate provided by the GSF
+	uint64_t EKFGSF_yaw_reset_time_ms{0};	// timestamp of last emergency yaw reset (uSec)
```

Comment:
no need to init these variables, they will be zero on startup automatically. Using init just adds flash space for no reason